src/bun.js/net.exports.js

```diff
@@ -79,7 +80,7 @@ export const Socket = (function (InternalSocket) {
   );
 })(
   class Socket extends Duplex {
-    static #Handlers = {
+    static _Handlers = {
```

Comment:
This should not be exposed publicly. If it needs to be exposed, use a `Symbol` for the name instead of an identifier

Reply:
done on https://github.com/oven-sh/bun/pull/2337/commits/dc6bc3c6d8d73d3f5588116eeeaab409d01c765f